chromium_src/content/browser/renderer_host/render_frame_host_impl.cc

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "src/content/browser/renderer_host/render_frame_host_impl.cc"
+
+namespace content {
+
+void RenderFrameHostImpl::GetImageAt(
+    int x,
+    int y,
+    base::OnceCallback<void(const SkBitmap&)> callback) {
+  gfx::PointF point_in_view =
+      GetView()->TransformRootPointToViewCoordSpace(gfx::PointF(x, y));
+  GetAssociatedLocalFrame()->GetImageAt(
+      gfx::Point(point_in_view.x(), point_in_view.y()), std::move(callback));
```

Comment:
it's possible to trigger this mojo call more than once if a renderer is busy (a slow machine or just a heavy JS is currently running).

pls make sure we won't do anything weird like crash/show multiple dialogs, etc..